### PR TITLE
EZP-28954: Label is one line below checkbox/radiobutton

### PR DESCRIPTION
**Pull request by webhdx**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-28954
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/config/views.yml [hunk 1]
```diff
@@ -27,3 +27,5 @@ system:
                     match: true
                     params:
                         viewbaseLayout: '@EzPlatformAdminUi/layout.html.twig'
+        fielddefinition_edit_templates:
```
Comment (mikadamczyk) [inline src/bundle/Resources/config/views.yml]: `field_definition_edit_templates` ?

File: src/bundle/Resources/views/admin/content_type/edit.html.twig [hunk 2]
```diff
@@ -100,6 +100,7 @@
                             {% for child in field_definition %}
                                 {# Iterable children means collections, they are handled in field_types.html.twig #}
                                 {% if not child.rendered and child is not iterable %}
+                                {{ dump(child) }}
```
Comment (mikadamczyk) [inline src/bundle/Resources/views/admin/content_type/edit.html.twig]: please, remove `dump()`

File: src/bundle/Resources/config/views.yml (hunk 1 above)
Comment (webhdx) [reply src/bundle/Resources/config/views.yml]: Nope, this is proper name defined in the kernel.

Comment (mikadamczyk) [reply src/bundle/Resources/config/views.yml]: ok, sorry :)

File: src/bundle/Resources/views/admin/content_type/field_types.html.twig [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+{% block ezobjectrelation_field_definition_edit %}
+    <div class="ezobjectrelation-settings selection-root">
+        {{- form_row(form.selectionRoot) -}}
+
+        <div class="mt-1">
+            <button data-universaldiscovery-title="{{"field_definition.ezobjectrelation.selection_root_udw_title"|trans({}, "ezrepoforms_content_type")}}"
+                    class="ez-button-tree pure-button ez-font-icon ez-button ez-relation-pick-root-button btn btn-secondary btn--udw-relation-default-location"
+                    data-relation-root-input-selector="#{{form.selectionRoot.vars.id}}"
+                    data-relation-selected-root-name-selector="#{{form.selectionRoot.vars.id}}-selected-root-name"
+                    data-starting-location-id="1"
```
Comment (dew326) [inline src/bundle/Resources/views/admin/content_type/field_types.html.twig]: hardcoded `1`, is it ok?

Comment (webhdx) [reply src/bundle/Resources/views/admin/content_type/field_types.html.twig]: Definitely not.

File: src/bundle/Resources/views/admin/content_type/field_types.html.twig [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+
+    <div class="ezselection-settings options{% if group_class is not empty %} {{ group_class }}{% endif %}">
+        {{ form_label(form.options) }}
+        {{ form_errors(form.options) }}
+        <ul class="ezselection-settings-option-list ez-selection-table list-unstyled" data-selection-buttons=".ezselection-settings-option-remove">
+        {% for option in form.options %}
+            <li class="ezselection-settings-option-value ez-selection-table-row mb-2 d-flex align-items-center">
+                <input type="checkbox" class="ezselection-settings-option-checkbox ez-selection-table-checkbox">
+                {{ form_errors(option) }}
+                {{ form_widget(option, { 'attr': { 'class': 'form-contro ml-1 mb-0'}}) }}
```
Comment (dew326) [inline src/bundle/Resources/views/admin/content_type/field_types.html.twig]: `form-control`? (missing `l`)

File: src/bundle/Resources/views/admin/content_type/field_types.html.twig [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+            </li>
+        {% endfor %}
+        </ul>
+        <script type="text/template" class="ezselection-settings-option-value-prototype" data-ezselection-option-input-name="{{ form.options.vars.prototype.vars.full_name }}" data-ezselection-option-input-id="{{ form.options.vars.prototype.vars.id }}">
+            <li class="ezselection-settings-option-value ez-selection-table-row mb-2 d-flex align-items-center">
+                <input type="checkbox" class="ezselection-settings-option-checkbox ez-selection-table-checkbox">
+                {{ form_widget(form.options.vars.prototype, { 'attr': { 'class': 'form-contro ml-1 mb-0'}}) }}
```
Comment (dew326) [inline src/bundle/Resources/views/admin/content_type/field_types.html.twig]: `l`?

**Comment by webhdx** [comment]
All fixed.

**Pull request merged**